xbmc/pvr/epg/EpgDatabase.cpp

```diff
@@ -142,6 +141,105 @@ void CPVREpgDatabase::UpdateTables(int iVersion)
   }
+
+  if(iVersion < 13)
+  {
+      m_pDS->exec(
+          "CREATE TABLE epgtags_new ("
+          "idBroadcast     integer primary key, "
+          "iBroadcastUid   integer, "
+          "idEpg           integer, "
+          "sTitle          varchar(128), "
+          "sPlotOutline    text, "
+          "sPlot           text, "
+          "sOriginalTitle  varchar(128), "
+          "sCast           varchar(255), "
+          "sDirector       varchar(255), "
+          "sWriter         varchar(255), "
+          "iYear           integer, "
+          "sIMDBNumber     varchar(50), "
+          "sIconPath       varchar(255), "
+          "iStartTime      integer, "
+          "iEndTime        integer, "
+          "iGenreType      integer, "
+          "iGenreSubType   integer, "
+          "sGenre          varchar(128), "
+          "sFirstAired     varchar(32), "
+          "iParentalRating integer, "
+          "iStarRating     integer, "
+          "iSeriesId       integer, "
+          "iEpisodeId      integer, "
+          "iEpisodePart    integer, "
+          "sEpisodeName    varchar(128), "
+          "iFlags          integer, "
+          "sSeriesLink     varchar(255)"
+          ")"
+      );
+
+      m_pDS->exec(
+          "INSERT INTO epgtags_new ("
+          "idBroadcast, "
+          "iBroadcastUid, "
+          "idEpg, "
+          "sTitle, "
+          "sPlotOutline, "
+          "sPlot, "
+          "sOriginalTitle, "
+          "sCast, "
+          "sDirector, "
+          "sWriter, "
+          "iYear, "
+          "sIMDBNumber, "
+          "sIconPath, "
+          "iStartTime, "
+          "iEndTime, "
+          "iGenreType, "
+          "iGenreSubType, "
+          "sGenre, "
+          "sFirstAired, "
+          "iParentalRating, "
+          "iStarRating, "
+          "iSeriesId, "
+          "iEpisodeId, "
+          "iEpisodePart, "
+          "sEpisodeName, "
+          "iFlags, "
+          "sSeriesLink"
+          ") "
+          "SELECT "
+          "idBroadcast, "
+          "iBroadcastUid, "
+          "idEpg, "
+          "sTitle, "
+          "sPlotOutline, "
+          "sPlot, "
+          "sOriginalTitle, "
+          "sCast, "
+          "sDirector, "
+          "sWriter, "
+          "iYear, "
+          "sIMDBNumber, "
+          "sIconPath, "
+          "iStartTime, "
+          "iEndTime, "
+          "iGenreType, "
+          "iGenreSubType, "
+          "sGenre, "
+          "CASE WHEN iFirstAired <= 0 THEN '' ELSE COALESCE(STRFTIME('%Y-%m-%d', iFirstAired, 'UNIXEPOCH'), '') END AS sFirstAired, "
+          "iParentalRating, "
+          "iStarRating, "
+          "iSeriesId, "
+          "iEpisodeId, "
+          "iEpisodePart, "
+          "sEpisodeName, "
+          "iFlags, "
+          "sSeriesLink "
+          "FROM epgtags"
+      );
+
+      m_pDS->exec("DROP TABLE epgtags");
+      m_pDS->exec("ALTER TABLE epgtags_new RENAME TO epgtags");
```

Comment:
I did not, great catch thank you.  It will need a secondary UPDATE query after the initial SELECT INTO as the data conversion method differs between the two databases.  Will update.